packages/appstore/src/modules/tour-guide/tour-guide.tsx

```diff
@@ -136,7 +126,15 @@ const TourGuide = () => {
             floaterProps={{
                 disableAnimation: true,
             }}
-            callback={data => setJoyrideIndex(data.index)}
+            callback={data => {
+                setJoyrideIndex(data.index);
+                if (data.action === 'prev' && data.index === tour_step_config_high_risk.length) {
+                    toggleIsTourOpen(false);
+                    setTimeout(() => {
+                        toggleIsTourOpen(true);
+                    }, 0);
+                }
+            }}
```

Comment:
There's a better way to handle this based on the joyride config file. In my opinion this implementation might solve whatever issue we might facing right now but it will introduce more if we ever decide on changing the onboarding content.